src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -230,4 +244,127 @@ export class SignifyApi {
     const allIdentifiersResult = await this.signifyClient.identifiers().list();
     return allIdentifiersResult;
   }
+
+  async createMultisig(
+    aid: Aid,
+    otherAids: Pick<Aid, "state">[]
+  ): Promise<{
+    op: any;
+    icpResult: EventResult;
+    name: string;
+  }> {
+    const rstates = [aid["state"], ...otherAids.map((aid) => aid["state"])];
+    const states = rstates;
```

Comment:
If `states` always equals `rstates`, can we just use that everywhere and not have the extra variable?